exercises/webapp/src/components/NewsItem.vue

```diff
@@ -1,39 +1,39 @@
 <template>
   <div class="news-item">
-    <h3>{{ newsItem.title }} ({{ newsItem.voteCount }})</h3>
+    <h2>{{ newsItem.title }} ({{ newsItem.voteCount }})</h2>
     <span>
       <button @click="upvote">upvote</button>
       <button @click="downvote">downvote</button>
-      <button @click="remove">remove</button>
+      <button @click="remove" class="remove-button">remove</button>
     </span>
   </div>
 </template>
 
 <script>
 export default {
-  name: 'NewsItem',
-  props: ['newsItem'],
+  name: "NewsItem",
+  props: ["newsItem"],
   data() {
-    return {}
+    return {};
   },
   methods: {
     upvote() {
-      this.$emit('update', {
+      this.$emit("update", {
         newsItem: this.newsItem,
         voteChange: 1
       });
     },
     downvote() {
-      this.$emit('update', {
+      this.$emit("update", {
```

Comment:
My linter always asks me to use single quotation.

Reply:
It depends on the configuration in `.eslintrc.js`